app/models/user.rb

```diff
@@ -15,13 +15,16 @@
 #  index_users_on_type_and_email  (type,email) UNIQUE
 #
 class User < ApplicationRecord
-  has_many :memberships
+  SUPPORT_EMAIL_REGEXP = /\A[a-zA-Z0-9.!\#$%&'*+\/=?^_`{|}~-]+@education.gov.uk/
+
+  has_many :memberships, dependent: :destroy
 
   validates :first_name, presence: true
   validates :last_name, presence: true
   validates :email, presence: true, format: { with: URI::MailTo::EMAIL_REGEXP }
   validates :type, presence: true
   validates :email, uniqueness: { scope: :type, case_sensitive: false }
+  validates :email, format: { with: SUPPORT_EMAIL_REGEXP, message: :invalid_support_email }, if: :support_user?
```

Comment:
Should this logic be in the `ActsAsSupportUser` concern, as this is only relevant to SupportUsers.

Reply:
Sounds good :)

Reply:
Addressed in https://github.com/DFE-Digital/itt-mentor-services/pull/135/commits/a688a17652a1abc5964759bc7d9b022fa5aaa59c